projects/1.20/assets/bovines-and-buttercups/bovinesandbuttercups/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.title": "状态免除",
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.description": "使用花蜜避免状态效果的影响。",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.title": "状态锁定",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.description": "使用花蜜锁定状态效果的持续时间。",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.title": "斑斓银河",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.description": "繁殖出变异的哞花子代。",
+    
+    "block.bovinesandbuttercups.buttercup": "毛茛",
+    "block.bovinesandbuttercups.pink_daisy": "粉色雏菊",
+    "block.bovinesandbuttercups.limelight": "水亚木",
+    "block.bovinesandbuttercups.bird_of_paradise": "天堂鸟",
+    "block.bovinesandbuttercups.chargelily": "青百合",
+    "block.bovinesandbuttercups.hyacinth": "风信子",
+    "block.bovinesandbuttercups.snowdrop": "雪花莲",

```

```suggestion
    "block.bovinesandbuttercups.snowdrop": "雪滴花",
```
https://en.wikipedia.org/wiki/Galanthus
https://www.iplant.cn/info/Galanthus